### PR TITLE
Css

### PR DESCRIPTION
**Pull request by kbuechner**
I'm surprised that there's no conflicts but I guess I just added stuff & didn't edit any existing material? The things I wrote don't BREAK anything and I'd dig a look-over to see how things mesh. There may be redundancies. 

Sorry it's called CSS, wasn't expecting to dig so deep into server stuff and more angular

Still gotta make that angular form for reviews but it'll be fairly straightforward. Next step is making it specific to authenticated users!

### CODE REVIEW

File: browser/js/catalog/product.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 					{{ product.price | currency }}
 				</h4>
 
-				<img ng-src = "{{ product.photoUrl }}">
+				<img ng-src = "{{ product.photoUrl }}"  height= 80px width= 80px>
```
Comment (kbuechner) [inline browser/js/catalog/product.html:11]: temp until we css it up

File: browser/js/common/factories/review.factory.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+app.factory('ReviewFactory', function ($http, 'ProductFactory', 'UserFactory') {
```
Comment (tornglintaffychen) [inline browser/js/common/factories/review.factory.js]: don't need to put factory in quote.

File: browser/js/common/factories/review.factory.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+app.factory('ReviewFactory', function ($http, ProductFactory, UserFactory) {
```
Comment (tornglintaffychen) [inline browser/js/common/factories/review.factory.js:1]: don't need quotation marks around the ProductFactory and UserFactory, removed.

File: browser/js/common/factories/review.factory.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        .then(function(res){
+            console.log('here is the delete res: ', res)
+        })
+        .catch(function(error){
+            console.error("somethingn went wrong with deleting your review! ", error)
+        })
+    }
+    return ReviewFactory;
+
+}
```
Comment (tornglintaffychen) [inline browser/js/common/factories/review.factory.js]: missing ) after }

File: browser/js/common/directives/review/review.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+app.controller("ReviewController", function($scope, UserFactory) {
+  $scope.getUser = function(id){
+  	return UserFactory.fetchById()
+  	.then(function(foundUser){
+  		return foundUser.email;
+  	})
+  	.catch(function(error){
+  		console.error("something went wrong with fetching the user email! ", error)
+  	})
+  }
```
Comment (maxww) [inline browser/js/common/directives/review/review.js]: missing ) after }

File: browser/js/common/directives/review/review.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+app.controller("ReviewController", function($scope, UserFactory) {
+  $scope.getUser = function(id){
+  	return UserFactory.fetchById()
+  	.then(function(foundUser){
+  		return foundUser.email;
+  	})
+  	.catch(function(error){
+  		console.error("something went wrong with fetching the user email! ", error)
+  	})
+  })
```
Comment (tornglintaffychen) [inline browser/js/common/directives/review/review.js]: was missing ) after }, added.

File: browser/js/common/factories/review.factory.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        .then(function(res){
+            console.log('here is the delete res: ', res)
+        })
+        .catch(function(error){
+            console.error("somethingn went wrong with deleting your review! ", error)
+        })
+    }
+    return ReviewFactory;
+
+})
```
Comment (tornglintaffychen) [inline browser/js/common/factories/review.factory.js:45]: was missing ) after }, added.

File: browser/js/common/directives/review/review.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+  $scope.getUser = function(id){
+  	return UserFactory.fetchById()
+  	.then(function(foundUser){
+  		return foundUser.email;
+  	})
+  	.catch(function(error){
+  		console.error("something went wrong with fetching the user email! ", error)
+  	})
+  }
+})
```
Comment (tornglintaffychen) [inline browser/js/common/directives/review/review.js:11]: missing }), added

**Pull request merged**